src/ui/components/Header/index.tsx

```diff
@@ -9,8 +11,15 @@ import { useWallet } from "@src/ui/hooks/wallet";
 
 import "./header.scss";
 
+const METAMASK_INSTALL_URL = "https://metamask.io/";
+
 export const Header = (): JSX.Element => {
-  const { address, isActivating, isActive, chain, onConnect, onDisconnect, onLock } = useWallet();
+  const { address, isActivating, isActive, isInjectedWallet, chain, onConnect, onDisconnect, onLock } = useWallet();
+  const isLoading = isActivating && isInjectedWallet;
+
+  const onGoToMetamaskPage = useCallback(() => {
+    browser.tabs.create({ url: METAMASK_INSTALL_URL });
```

Comment:
I'm wondering why we didn't use `redirectToNewTab()` here? `src/util/browser.ts`